groups/prometheus/variables.tf

```diff
@@ -91,12 +86,19 @@ variable "vault_password" {
   description = "Password for connecting to Vault - usually supplied through TF_VARS"
 }
 
-variable "dns_zone_id" {
-  type  = string
-  description = "The ID of the zone to be used"
-}
-
 variable "dns_zone_name" {
   type = string
   description = "The zone name to be used"
 }
+
+variable "vpc_name" {
+  type        = string
+  description = "The name of the VPC to deploy in to"
+  default     = "management-vpc"
+}
+
+variable "subnet_pattern" {
```

Comment:
This should probably be sourced from Vault given it's a public repository

Reply:
Updated in [a19b2e1](https://github.com/companieshouse/prometheus-terraform/pull/15/commits/a19b2e1c9dce1c5750aec38a2e4b5fa296e79cd6)